README.md

```diff
@@ -129,8 +129,7 @@ See the [API documentation](http://esri.github.io/cedar/api) for further details
 Make sure you have the [Grunt CLI](http://gruntjs.com/getting-started) installed.
 
 1. `cd` into the `cedar` folder
-1. Install the dependencies with `npm install`
-1. Install additional dependencies with `bower install` (if you encounter an error connecting to github take a look at [this thread](https://github.com/angular/angular-phonecat/issues/141) for a possible fix).
+1. Install the dependencies with `npm install` (if you encounter an error connecting to github take a look at [this thread](https://github.com/angular/angular-phonecat/issues/141) for a possible fix).
```

Comment:
you can remove the bit about encountering the error - that only applies to bower installs